docs/source/en/using-diffusers/workflows.md

```diff
@@ -0,0 +1,331 @@
+             ('solver_order', 2),
+             ('prediction_type', 'epsilon'),
+             ('thresholding', False),
+             ('dynamic_thresholding_ratio', 0.995),
+             ('sample_max_value', 1.0),
+             ('predict_x0', True),
+             ('solver_type', 'bh2'),
+             ('lower_order_final', True),
+             ('disable_corrector', []),
+             ('solver_p', None),
+             ('use_karras_sigmas', False),
+             ('timestep_spacing', 'linspace'),
+             ('steps_offset', 1),
+             ('_use_default_values',
+              ['lower_order_final',
+               'sample_max_value',
+               'solver_p',
+               'dynamic_thresholding_ratio',
+               'thresholding',
+               'solver_type',
+               'prediction_type',
+               'predict_x0',
+               'use_karras_sigmas',
+               'disable_corrector',
+               'timestep_spacing',
+               'solver_order']),
+             ('skip_prk_steps', True),
+             ('set_alpha_to_one', False),
+             ('_class_name', 'PNDMScheduler'),
+             ('_diffusers_version', '0.6.0'),
+             ('clip_sample', False)])}
+```
+
+As you can notice, the `image` passed to the `pipeline` isn't a part of `workflow`.
+
+Let's serialize it and reload the pipeline:
+
+```python
+workflow.save_workflow("my-simple-workflow-sd", filename="controlnet_simple.json", push_to_hub=True)
+```
+
+Steps to load the workflow into [`StableDiffusionControlNetPipeline`] should remain the same as the example above:
+
+```python
+# load control net and stable diffusion v1-5
+controlnet = ControlNetModel.from_pretrained("lllyasviel/sd-controlnet-canny", torch_dtype=torch.float16)
+pipe = StableDiffusionControlNetPipeline.from_pretrained(
+    "runwayml/stable-diffusion-v1-5", controlnet=controlnet, torch_dtype=torch.float16
+)
+pipe.enable_model_cpu_offload()
+
+pipe.load_workflow("sayakpaul/my-simple-workflow-sd", filename="controlnet_simple.json")
+```
+
+If you try to generate an image now, it'll return the following error:
+
+```bash
+TypeError: image must be passed and be one of PIL image, numpy array, torch tensor, list of PIL images, list of numpy arrays or list of torch tensors, but is <class 'NoneType'>
+```
+
+To resolve the error, manually pass the conditioning image `canny_image`:
+
+```python
+image = pipe(image=canny_image).images[0]
+```
+
+Other unsupported serialization types include:
+
+* LoRA checkpoints: any information from LoRA checkpoints that might be loaded into a pipeline isn't serialized. Workflows generated from pipelines loaded with a LoRA checkpoint should be handled cautiously! You should ensure the LoRA checkpoint is loaded into the pipeline first before loading the corresponding workflow.
+* Call arguments including the following types: `torch.Tensor`, `np.ndarray`, `Callable`, `PIL.Image.Image`, and `List[PIL.Image.Image]`. 
+
+## Workflow
```

Comment:
This could be split off into a separate `Workflow` API page in the Internal classes section :)

Reply:
But workflows should be regular classes. I will split it off.